latch_cli/services/cp/utils.py

```diff
@@ -0,0 +1,199 @@
+import os
```

Comment:
This file has a lot of stuff going on in it -- id split out the path stuff, the queries, and the auth header. Rest of the stuff seems like more util style utils.